styles/markdown.css

```diff
@@ -34,6 +34,17 @@ main {
     @apply text-base;
   }
 
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    &[id] a {
+      @apply text-neutral-900 dark:text-white;
```

Comment:
FYI for Tailwind we use one token per line: (per collab guides)

So it would look like:

```
@apply text-neutral-900
  dark:text-white;
```